packages/js/README.md

```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
-   */
-  ringtoneFile: './sounds/incoming_call.mp3',
-  /*
-    ringbackFile - Used when you disable Generate Ringback Tone to provide your own ringback sound.
-  */
-  ringbackFile: './sounds/ringback_tone.mp3',
+  /* or use your Connection credentials */
+  // login: username,
+  // password: password,
 });
 
+// Connect and login
+client.connect();
+
+// You can disconnect when you're done
+//  client.disconnect();
+```
+
+> See [TelnyxRTC#constructor](./docs/ts/classes/telnyxrtc.md#constructors) for all options.
+
+Important: You should treat Connection credentials as sensitive data and should not hardcode credentials into your frontend web application. Check out the [examples](./examples) for sample React code that handles username and password by prompting the user.
+
+To hear/view calls in the browser, you'll need to specify an HTML media element:
+
+```js
+client.remoteElement = 'remoteMedia';
+```
+
+The corresponding HTML:
+
+```html
+<audio id="remoteMedia" autoplay="true" />
+<!-- or for video: -->
+<!-- <video id="remoteMedia" autoplay="true" playsinline="true" /> -->
+```
+
+### Events
+
+```js
 // Create a variable to track the current call
 let activeCall;
 
 // Attach event listeners
 client
-  .on('telnyx.socket.open', () => console.log('socket open'))
-  .on('telnyx.socket.close', () => {
-    console.log('socket closed');
-    client.disconnect();
-  })
-  .on('telnyx.socket.error', (error) => {
-    console.log('telnyx.socket.error', error);
-    client.disconnect();
-  })
   .on('telnyx.ready', () => console.log('ready to call'))
   .on('telnyx.error', () => console.log('error'))
-  // Event fired on call updates, e.g. when there's an incoming call
+  // Events are fired on both session and call updates
+  // ex: when the session has been established
+  // ex: when there's an incoming call
   .on('telnyx.notification', (notification) => {
-    activeCall = notification.call;
-
-    switch (notification.type) {
-      case 'callUpdate':
-        // Call is over and can be removed
-        if (
-          notification.call.state === 'hangup' ||
-          notification.call.state === 'destroy'
-        ) {
-          activeCall = null;
-        }
-        // An established and active call
-        if (notification.call.state === 'active') {
-          return;
-        }
-        // New calls that haven't started connecting yet
-        if (notification.call.state === 'new') {
-          return;
-        }
-        // Receiving an inbound call
-        if (notification.call.state === 'ringing') {
-          return;
-        }
-        // Call is active but on hold
-        if (notification.call.state === 'held') {
-          return;
-        }
-        break;
+    if (notification.type === 'callUpdate') {
+      activeCall = notification.call;
     }
   });
-
-// Connect and login
-client.connect();
-
-// You can disconnect when you're done
-// client.disconnect();
```

Comment:
I think we should keep this example, this is easier to customer see what he can do with our SDK.

@hugobdev any thoughts?

Reply:
I think that we can keep a simpler readme as Sua proposed, and guide the user to see the full-blown example as they are actually integrating with the SDK.